tests/functional/conftest.py

```diff
@@ -170,6 +147,18 @@ def pytest_runtest_teardown():
     exception_catcher("teardown", outcome)
 
 
+@pytest.hookimpl(hookwrapper=True)
+def pytest_runtest_teardown(item):
```

Comment:
Item is arg name from pytest API spec:
https://docs.pytest.org/en/6.2.x/reference.html

We could rename it, but it will confuse devs.